File: Swift/Sort/11651.swift
let N = Int(readLine()!)!
var arr = [[Int]]()
for _ in 1...N {
    let inputs = readLine()!.split(separator: " ").map{ Int($0)! }
    arr.append(inputs)
}

arr = arr.sorted(by: { ($0[1], $0[0]) < ($1[1], $1[0]) })
for i in 0...N-1 {
    print(arr[i].map{ String($0) }.joined(separator: " ") )
}

File: Python/11651.py
import sys

N = int(sys.stdin.readline())
arr = []

for _ in range(N):
    arr.append(list(map(int, sys.stdin.readline().split())))
ans = sorted(arr, key = lambda x: (x[1], x[0]))
for a in ans:
    print(" ".join(map(str, a)))
